flags/14.0/vttestserver.txt

```diff
@@ -1,184 +1,145 @@
-      --stats_combine_dimensions string                                       List of dimensions to be combined into a single "all" value in exported stats vars
-      --stats_common_tags string                                              Comma-separated list of common tags for the stats backend. It provides both label and values. Example: label1:value1,label2:value2
-      --stats_drop_variables string                                           Variables to be dropped from the list of exported variables.
-      --stats_emit_period duration                                            Interval between emitting stats to all registered backends (default 1m0s)
-      --stderrthreshold value                                                 logs at or above this threshold go to stderr (default 1)
-      --tablet_dir string                                                     The directory within the vtdataroot to store vttablet/mysql files. Defaults to being generated by the tablet uid.
-      --tablet_hostname string                                                The hostname to use for the tablet otherwise it will be derived from OS' hostname (default "localhost")
-      --tablet_manager_grpc_ca string                                         the server ca to use to validate servers when connecting
-      --tablet_manager_grpc_cert string                                       the cert to use to connect
-      --tablet_manager_grpc_concurrency int                                   concurrency to use to talk to a vttablet server for performance-sensitive RPCs (like ExecuteFetchAs{Dba,AllPrivs,App}) (default 8)
-      --tablet_manager_grpc_connpool_size int                                 number of tablets to keep tmclient connections open to (default 100)
-      --tablet_manager_grpc_crl string                                        the server crl to use to validate server certificates when connecting
-      --tablet_manager_grpc_key string                                        the key to use to connect
-      --tablet_manager_grpc_server_name string                                the server name to use to validate server certificate
-      --tablet_manager_protocol string                                        the protocol to use to talk to vttablet (default "grpc")
-      --topo_consul_lock_delay duration                                       LockDelay for consul session. (default 15s)
-      --topo_consul_lock_session_checks string                                List of checks for consul session. (default "serfHealth")
-      --topo_consul_lock_session_ttl string                                   TTL for consul session.
-      --topo_consul_watch_poll_duration duration                              time of the long poll for watch queries. (default 30s)
-      --topo_etcd_lease_ttl int                                               Lease TTL for locks and leader election. The client will use KeepAlive to keep the lease going. (default 30)
```

Comment:
we need these?

Reply:
As far as I am reading the code vttestserver does hold reference to topo server but its never make a call itself or act as  a client to topo server. so i don't think so we need these flags... @deepthi  can you confirm though..